server/realTime/scripts/Network.js

```diff
@@ -19,6 +19,7 @@ var BIG_BRUSH_MIN_REP = 5;
 var MEMBER_MIN_REP = 15;
 var UPVOTE_MIN_REP = 7;                  // Has to be changed in the playerserver too
 var SHARE_IP_MIN_REP = MEMBER_MIN_REP;
+var REGION_MIN_REP = 25;
```

Comment:
Given the fact that there is currently no size limit yet. I think I'd rather have a higher number. Unless we first add restrictions.